.gitignore

```diff
@@ -1 +1,7 @@
 /bin/
+.idea
+.openshift_install*
```

Comment:
this should only include installer'd build artifacts and therefore bin exists.

install create assets etc... should not be part of the public gitignore.